cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -233,28 +230,29 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
         }
 
         /**
-         * Asynchronously reconciles the cluster CA secret.
-         * The secret has have the name determined by {@link AbstractModel#getClusterCaName(String)}.
-         * Within the secret the current certificate is stored under the key {@code cluster-ca.crt}
-         * and the current key is stored under the key {@code cluster-ca.crt}.
-         * Old certificates which are still within their validity are stored under keys named like {@code cluster-ca-<not-after-date>.crt}, where
+         * Asynchronously reconciles the cluster and clients CA secrets.
+         * The cluster CA secret has to have the name determined by {@link AbstractModel#clusterCaSecretName(String)}.
+         * The clients CA secret has to have the name determined by {@link KafkaCluster#clientsCaSecretName(String)}.
+         * Within both the secrets the current certificate is stored under the key {@code ca.crt}
+         * and the current key is stored under the key {@code ca.key}.
+         * Old certificates which are still within their validity are stored under keys named like {@code ca-<not-after-date>.crt}, where
```

Comment:
From this line on the Javadoc is incorrect: We currently only ever store a single `ca.crt` in the secret.

As an aside: I've been thinking about removing all the code for supporting having multiple certs in the Secret. The reason I haven't is that I think we will need it eventually to support CA replacement (where we replace both the cert and the key).